### PR TITLE
README for nginx router: Steps to start nginx router on openshift

### PR DESCRIPTION
**Pull request by rajatchopra**
Replaces #16869 

@openshift/networking

### CODE REVIEW

File: images/router/nginx/README.md [hunk 1]
```diff
@@ -0,0 +1,41 @@
+### 1. Build the image
+In nodes,build origin openshif/nginx-router image:
```
Comment (pravisankar) [inline images/router/nginx/README.md]: *openshift

File: images/router/nginx/README.md [hunk 2]
```diff
@@ -0,0 +1,41 @@
...
+ - git clone https://github.com/openshift/origin
+ - cd origin/images/router/nginx/
+ - docker build -t openshift/nginx-router .
+
+### 2. Run the nginx router
+In master, deploy and edit nginx-router so that working directory can be fixed
+ - oadm policy add-scc-to-user hostnetwork -z router
```
Comment (pravisankar) [inline images/router/nginx/README.md]: Don't we need this below policy, similar to HAProxy?
`oadm policy add-cluster-role-to-user cluster-reader system:serviceaccount:default:router`

File: images/router/nginx/README.md [hunk 3]
```diff
@@ -0,0 +1,41 @@
...
+
+### 3. Test
+Test no security route in nginx-router
+ - curl https://raw.githubusercontent.com/openshift/origin/master/examples/hello-openshift/hello-pod.json | oc create -f -
+ - oc expose pod hello-openshift
+ - oc expose svc hello-openshift
+ - oc get pod -o wide
+```
+NAME READY STATUS RESTARTS AGE IP NODE
+hello-openshift 1/1 Running 0 5m 172.21.0.7 host-8-241-64.host.centralci.eng.rdu2.redhat.com
```
Comment (pweil-) [inline images/router/nginx/README.md]: probably want to remove these internal hosts

File: images/router/nginx/README.md [hunk 4]
```diff
@@ -0,0 +1,41 @@
...
+In nodes,build origin openshif/nginx-router image:
+ - git clone https://github.com/openshift/origin
+ - cd origin/images/router/nginx/
+ - docker build -t openshift/nginx-router .
+
+### 2. Run the nginx router
+In master, deploy and edit nginx-router so that working directory can be fixed
+ - oadm policy add-scc-to-user hostnetwork -z router
+ - oadm router nginx-router --images=openshift/nginx-router
+ - oc edit deploymentconfigs/nginx-router -o json
```
Comment (php-coder) [inline images/router/nginx/README.md]: Consider using `oc patch` here.

**Pull request closed without merging**